### PR TITLE
Use `hd-wallet` instead of `slip-10`

### PR DESCRIPTION
**Pull request by survived**

### CODE REVIEW

**Comment by survived** [comment]
I have no idea what is going on with docs in CI

File: Cargo.toml [hunk 1]
```diff
@@ -41,6 +41,6 @@ serde_with = { version = "2", default-features = false }
 serde_json = "1"
 hex = { version = "0.4", default-features = false }
 
-slip-10 = { version = "0.4", default-features = false }
+hd-wallet = { version = "0.5", git = "https://github.com/LFDT-Lockness/slip-10", branch = "eddsa-support", default-features = false }
```
Comment (survived) [inline Cargo.toml]: to be published first

**Comment by maurges** [comment]
Looks good

**Comment by survived** [comment]
@maurges I updated the deps

**Review by maurges — APPROVED** [review]

**Pull request merged**